### DISCUSSION TITLE
Material List Custom sorting

### DISCUSSION THREAD

**Discussion by Redostrike** [maruohon/litematica#474] [Ideas]
Would it be possible or would there be an easy way to have the user customize (by editing a config file) to have the material list be sorted in a custom order instead? I'm not sure how it sorts now it seems to be somewhat most used blocks first but its not completely that way. This would help a lot for long term worlds having it that the list is ordered via your own storage system and getting materials would be way faster than sometimes running from one end to the other end.

**Comment by maruohon** [comment]
It's currently by default sorted based on the total number of items, I believe? And you can re-order it by any of the columns, by clicking on the column header/title.

But I can see your point, that would likely be pretty useful. I'll add it to my todo list.
